RenderStatePackager/src/ParsingEnvironment.cpp

```diff
@@ -143,7 +138,4 @@ bool ParsingEnvironment::Initilize()
     }
 }
 
-ParsingEnvironment::~ParsingEnvironment()
-{}
```

Comment:
There is a good reason to have destructor in cpp file. If it is not in the CPP, it becomes inline function and is compiled in every file that includes the header. Note that even when it is empty, the compiler still adds destructors for base class and all members.
It may fail to compile if you have something like this:

```cpp
class ForwardDeclared;

class MyClass
{
   strd::unique_ptr<ForwardDeclared> m_Ptr;
};
```

This will fail to compile if you don't include the definition `ForwardDeclared`